### ISSUE TITLE
Power amounts are shown as FE instead of E on Fabric

### ISSUE THREAD

**Issue by Technici4n** [Technici4n/Powah#5]

**Comment by warior456** [comment]
can't you make it rf it's more used even on forge

**Comment by 62832** [comment]
> can't you make it rf it's more used even on forge

Just read the issue title a few more times until you've understood.

**Comment by Technici4n** [comment]
I can add a config option that lets you choose the exact text (`FE`, `E`, `RF`, whatever). Is that what you're asking for @warior456?

**Comment by warior456** [comment]
> I can add a config option that lets you choose the exact text (`FE`, `E`, `RF`, whatever). Is that what you're asking for @warior456?

yes

**Comment by unix-supremacist** [comment]
yeah arbitrary energy name would be nice for packs trying to integrate with specific mods since TRE doesn't have a standard energy naming convention

**Comment by Technici4n** [comment]
Fabric support is discontinued in 1.21+.
